taqueria-plugin-smartpy/common.ts

```diff
@@ -0,0 +1,43 @@
+import { sendErr } from '@taqueria/node-sdk';
```

Comment:
Some functions in this file are very similar to the one in ligo plugin. Is there an easy way to unify them?

Reply:
I can put them in the SDK but they are only used for compilers. Doesn't seem necessary for this amount of code.